common/actions/config.js

```diff
@@ -1,41 +1,45 @@
 // @flow
+import {
+  NODE_CHANGE,
+  LANGUAGE_CHANGE,
+  GAS_PRICE_CHANGE
+} from './configConstants';
 
 export type ChangeNodeAction = {
-  type: 'CONFIG_NODE_CHANGE',
+  type: NODE_CHANGE,
```

Comment:
Flow's docs on redux, that I referenced: https://flow.org/en/docs/frameworks/redux/